keras_contrib/applications/densenet.py

```diff
@@ -323,3 +467,146 @@ def __create_dense_net(nb_classes, img_input, include_top, depth=40, nb_dense_bl
         x = Dense(nb_classes, activation='softmax', W_regularizer=l2(weight_decay), b_regularizer=l2(weight_decay))(x)
 
     return x
+
+
+def __create_fcn_dense_net(nb_classes, img_input, include_top, nb_dense_block=5, growth_rate=12,
```

Comment:
No. It is better to keep it separate. This is what I meant by merging them would cause problems. It is cleaner to keep two separate model building logic completely separate.